crates/workspace/src/persistence.rs

```diff
@@ -360,6 +363,9 @@ define_connection! {
         ALTER TABLE workspaces DROP COLUMN remote_project_id;
         ALTER TABLE workspaces ADD COLUMN dev_server_project_id INTEGER;
     ),
+    sql!(
+        ALTER TABLE workspaces ADD COLUMN local_paths_order BLOB;
```

Comment:
> BLOB

Note (no actions in this PR needed): this looks relatively inconvenient to read as a human, but all Zed workspace serialization system needs a refactor pass or two, so at this time might be ok.

Reply:
The serialisation could do with some docs too, even if it's just doc comments in the code ^^ I know all the docs for the code base are largely WIP since the Open Source move though.